airbyte-integrations/connectors/source-netsuite/source_netsuite/spec.yaml

```diff
@@ -0,0 +1,46 @@
+documentationUrl: https://docsurl.com
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: Netsuite Spec
+  type: object
+  required:
+    - realm
+    - consumer_key
+    - consumer_secret
+    - token_id
+    - token_secret
+    - start_datetime
+  additionalProperties: false
+  properties:
+    realm:
+      type: string
+      description: Netsuite realm e.g. 2344535 (followed by _SB# if it is a sandbox)
+      pattern: ^[0-9_SB]*
+    consumer_key:
+      type: string
+      description: Consumer key associated with your integration
+      airbyte_secret: true
+    consumer_secret:
+      type: string
+      description: Consumer secret associated with your integration
+      airbyte_secret: true
+    token_id:
+      type: string
+      description: Access token id
+      airbyte_secret: true
+    token_secret:
+      type: string
+      description: Access token secret
+      airbyte_secret: true
+    concurrency_limit:
+      type: integer
+      description: Maximum conccurent requests to be made to Netsuite. The default is for there to be no concurrency, but it is recommended as it speeds up the syncs significantly.
+    start_datetime:
+      type: string
+      description: Only get records updated after this date
+      pattern: ^(\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2})Z$
+    record_types:
+      type: array
+      items:
+        type: string
+      description: The API names of the Netsuite objects you want to sync. Setting this speeds up the connection setup process by limiting the number of schemas that need to be retrieved from Netsuite.
```

Comment:
so you need to specify what schema/streams do you want to sync? maybe give the option to sync aditional streams BUT the connector need the default ones.

Reply:
If you do not specify anything the connector will pull all objects present in the user's Netsuite instance. This takes a really long time, though, which is not a great experience and caused acceptance tests to fail.